### DISCUSSION TITLE
Show Block Nesting

### DISCUSSION THREAD

**Discussion by GeorgeAtArborSolutions** [codefori/vscode-ibmi#1346] [Ideas]
Source/Show Block Nesting is a favorite feature for dealing with poorly written legacy code to figure out just what conditions are present when a specific line of code is being executed.  Can't seem to find anything in Code that provides this kind of data and would love to see it implemented.

**Comment by Max-crosskey** [comment]
Yes this feature would be great, tried looking for some other extensions that might enable this but couldn't find any
